modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4030,7 +4030,8 @@ else if (sqlType == Types.INTEGER) {
 			preparedStatement.setInt(index, GetterUtil.getInteger(value));
 		}
 		else if (sqlType == Types.VARCHAR) {
-			preparedStatement.setString(index, String.valueOf(value));
+			preparedStatement.setString(
+				index, GetterUtil.getString(value, null));
```

Comment:
@p-albuquerque I've never seen `GetterUtil` being used with `null` as the default value.. 

Did you see some other example in our code? What if you do `(String)value` instead?